docs/hello_nextflow/03_hello_gatk.md

```diff
@@ -228,40 +222,54 @@ process GATK_HAPLOTYPECALLER {
 }
 ```
 
-#### 2.2. Add accessory inputs up top
+#### 2.2. Add definitions for accessory inputs
 
 ```groovy title="hello-gatk.nf"
 // Accessory files
-params.genome_reference = "${projectDir}/data/ref/ref.fasta"
-params.genome_reference_index = "${projectDir}/data/ref/ref.fasta.fai"
-params.genome_reference_dict = "${projectDir}/data/ref/ref.dict"
-params.calling_intervals = "${projectDir}/data/intervals.list"
+params.reference        = "${projectDir}/data/ref/ref.fasta"
+params.reference_index  = "${projectDir}/data/ref/ref.fasta.fai"
+params.reference_dict   = "${projectDir}/data/ref/ref.dict"
+params.intervals        = "${projectDir}/data/ref/intervals.bed"
+```
+
+#### 2.3. Make a value channel for each of the accessory files
+
+Add this to the workflow block (after the `reads_ch` creation):
+
+```groovy title="hello-gatk.nf"
+// Create channels for the accessory files (reference and intervals)
+ref_file        = file(params.reference)
+ref_index_file  = file(params.reference_index)
+ref_dict_file   = file(params.reference_dict)
+intervals_file  = file(params.intervals)
 ```
 
-#### 2.3. Add a call to the workflow block to run GATK_HAPLOTYPECALLER
+Here, the `collect()` operator converts the **queue channel** created by `Channel.fromPath()` to a **value channel**. See [docs](https://www.nextflow.io/docs/latest/channel.html#channel-types) for more details.
```

Comment:
Where is this `.collect()`?

Reply:
Oh it went away because Adam figured out a simpler syntax

Reply:
fixing this